src/transactions/reducer.ts

```diff
@@ -141,21 +148,11 @@ export const reducer = (
   }
 }
 
-function mapForContextId(
```

Comment:
i removed this because the `any` type on the `mapping` makes it very hard to work with.